_pages/policies.md

```diff
@@ -9,13 +9,15 @@ All StarCon attendees, speakers, volunteers, and sponsors are required to abide
 
 ## Photography Policy
 
-All StarCon attendees, volunteers, speakers, and sponsors will be provided with a badge. Three colours of badges are provided.
+All StarCon attendees, volunteers, speakers, and sponsors will be provided with a badge and a lanyard. Three colours of lanyards are provided.
 
-- Green means it is permitted for photographers to take pictures of you.
-- Blue means photographers are required to ask permission before taking pictures of you.
-- Red means photos should not be taken of you. (If any are taken by accident, they will not be published).
+- A **green** lanyard means that you allow our event photographer to take photographs of you.
```

Comment:
what about other people at the event? I figured it meant anyone could

Reply:
clarified it